values.yaml

```diff
@@ -97,9 +97,13 @@ securityContext: {}
 service:
   type: ClusterIP
   port: 3000
-
+  annotations:
```

Comment:
This does not allow multiple annotations (it throws a json parsing error)  so, for example, to put 
service.beta.kubernetes.io/aws-load-balancer-internal: "true" and 
service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
it fails.

Reply:
Wow. That's stupid. Because I put the {{ toyaml }} indented it complains. :roll_eyes:

Reply:
(I'm still fairly new to _creating_ helm charts so it's all trial by error for me too) 
Should be fixed now!

Reply:
No worries, you are ahead of me on this..lol